Panic in parseVisibilityAttr: could not match regexp

```
panic: could not match regexp with string
^(?P<address>[0-9a-fx]+) <(?P<position>.*)>(?:(?P<inherited> Inherited)?|(?P<default> Default)?)*[\s]*$
0x55fb731b4430 <line:24:16, col:35> Hidden
```

--- a/ast/visibility_attr.go
+++ b/ast/visibility_attr.go
@@ -7,6 +7,7 @@ type VisibilityAttr struct {
 	ChildNodes  []Node
 	IsDefault   bool
 	IsInherited bool
+	IsHidden    bool
 }
 
 func parseVisibilityAttr(line string) *VisibilityAttr {
@@ -14,6 +15,7 @@ func parseVisibilityAttr(line string) *VisibilityAttr {
 		`<(?P<position>.*)>
 		(?P<inherited> Inherited)?
 		(?P<default> Default)?
+		(?P<hidden> Hidden)?
 		`,
 		line,
 	)
@@ -24,6 +26,7 @@ func parseVisibilityAttr(line string) *VisibilityAttr {
 		ChildNodes:  []Node{},
 		IsDefault:   len(groups["default"]) > 0,
 		IsInherited: len(groups["inherited"]) > 0,
+		IsHidden:    len(groups["hidden"]) > 0,
 	}
 }
 

--- a/ast/visibility_attr_test.go
+++ b/ast/visibility_attr_test.go
@@ -12,6 +12,7 @@ func TestVisibilityAttr(t *testing.T) {
 			ChildNodes:  []Node{},
 			IsInherited: false,
 			IsDefault:   true,
+			IsHidden:    false,
 		},
 		`0x7f8e7b00bb80 </cmark/src/cmark.h:497:16, col:36> Inherited Default`: &VisibilityAttr{
 			Addr:        0x7f8e7b00bb80,
@@ -19,6 +20,15 @@ func TestVisibilityAttr(t *testing.T) {
 			ChildNodes:  []Node{},
 			IsInherited: true,
 			IsDefault:   true,
+			IsHidden:    false,
+		},
+		`0x55ab30581650 <line:24:16, col:35> Hidden`: &VisibilityAttr{
+			Addr:        0x55ab30581650,
+			Pos:         NewPositionFromString("line:24:16, col:35"),
+			ChildNodes:  []Node{},
+			IsInherited: false,
+			IsDefault:   false,
+			IsHidden:    true,
 		},
 	}
 

